.gitignore

```diff
@@ -1,16 +0,0 @@
-*.gz
```

Comment:
this file isn't specific to a build system.  you might want to trim a few entries for the GNU make setup, but deleting it entirely doesn't make sense.

Reply:
@vapier  This has been updated on master.